### ISSUE TITLE
What is the training dataset for RAVAESum?

### ISSUE THREAD

**Issue by iriscxy** [textmine/RA-MDS#2]
Did you just split the RA-MDS into training /validation/ test datasets or use a different dataset for training?

**Comment by lipiji** [comment]
Hi, The method is unsupervised. So we need not to split the dataset into training and testing. Actually, the basic model is from this AAAI paper: http://lipiji.com/docs/li2017salience.pdf .  I employ TAC 2010 as the validation set for the tuning work of some parameters.
